ephios/core/templates/core/event_list.html

```diff
@@ -33,9 +35,23 @@ <h1>{% translate "Events" %}
                            href="?{% param_replace mode="list" date=None %}"
                        {% endif %}
                     ><span class="fa fa-list"></span></a>
-                    <a class="btn btn-sm {% if mode == "calendar" %}btn-secondary disabled{% else %}btn-outline-secondary{% endif %}"
-                       href="?{% param_replace mode="calendar" %}"
-                    ><i class="fa fa-calendar"></i></a>
+
+                    <div class="btn-group" role="group">
+                        <button
+                            type="button"
+                            class="btn btn-sm dropdown-toggle {% if mode == "calendar" or mode == "day" %}btn-secondary{% else %}btn-outline-secondary{% endif %}"
+                            data-bs-toggle="dropdown"
+                            aria-expanded="false">
+                            <i class="fa fa-calendar"></i>
+                        </button>
+                        <ul class="dropdown-menu">
+                            <li><a class="dropdown-item {% if mode == "calendar" %} disabled {% endif %}"
+                                   href="?{% param_replace mode="calendar" %}">{% translate "Month" %}</a></li>
+                            <li><a class="dropdown-item {% if mode == "day" %} disabled {% endif %}"
+                                   href="?{% param_replace mode="day" %}">{% translate "Day" %}</a></li>
+                        </ul>
```

Comment:
@jeriox and me thought we do prefer the three buttons instead of the drop down for now. If we add more options in the future we can still revisit this.

Reply:
What icons would you want for the buttons?

Reply:
`<i class="fas fa-th"></i>` could work next to the existing ones. Or `<i class="fas fa-calendar-day"></i>` if that's distinguashable from the existing month calendar icon. Find more in [FontAwesomev5](https://fontawesome.com/v5/search?q=day&o=r)

Reply:
Those together with the current month icon would look like this

<img width="120" alt="Screenshot 2024-03-19 at 20 34 57" src="https://github.com/ephios-dev/ephios/assets/2546622/f57ce726-d303-4d37-b5ab-d3d8e803e715">
<img width="121" alt="Screenshot 2024-03-19 at 20 35 07" src="https://github.com/ephios-dev/ephios/assets/2546622/a8628e6f-b1fb-4057-81f7-6d671c901fcd">

I think using `calendar-day` for the day and `calendar-alt` is good at communicating the difference. What do you think?

<img width="126" alt="Screenshot 2024-03-19 at 20 36 01" src="https://github.com/ephios-dev/ephios/assets/2546622/2e104814-c12c-43db-b213-1bc86ba93555">